mm2src/mm2_rpc/src/data/legacy/activation.rs

```diff
@@ -24,3 +24,17 @@ pub struct CoinInitResponse {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub mature_confirmations: Option<u32>,
 }
+
+#[derive(Serialize, Deserialize)]
+#[serde(tag = "method", rename = "set_required_confirmations")]
+pub struct SetRequiredConfRequest {
```

Comment:
I had time trying to read/translate this so I guess this needs renaming e.g `SetRequiredConfirmationReq`

Reply:
Let me explain please, the idea is simple, I'm aiming to have a uniform rpc_data naming scheme and each type of it should end with "Request" or "Response"